src/axolotl/utils/models.py

```diff
@@ -343,12 +362,15 @@ def load_lora(model, cfg):
         PeftModel,
     )
 
-    lora_config = None
+    bits = 4 if cfg.load_in_4bits else 8 if cfg.load_in_8bits else None
```

Comment:
Simplify code to make it easier to read
```suggestion
    bits = None
    if cfg.cfg.load_in_4bits: 
        bits = 4
    elif cfg.cfg.load_in_8bits: 
        bits = 8
```